### PR TITLE
added test cases for OVA.py

### PR DESCRIPTION
**Pull request by samarthdd**
Added test cases for OVA.py

Found issue https://github.com/k8-proxy/k8-vmware/issues/11

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,4 +8,5 @@ pytest-cov
 coveralls
 #pytest-xdist       # didn't work (need to find out why)
 coverage
+wget
```
Comment (belwase) [inline requirements.txt:11]: let's try to make habit of adding versions of package too

File: tests/unit/vsphere/test_OVA.py [hunk 2]
```diff
@@ -69,4 +63,39 @@ def test_vms_ips(self):         # todo: move to Sdk to method called vms_ips() (
     #     photon_password = '{new pwd}}'
     #     vm_process = VM_Process(self.vm)
     #     vm_process.set_vm_account(photon_username, photon_password)
-    #     pprint(vm_process.ls("/"))
\ No newline at end of file
+    #     pprint(vm_process.ls("/"))
+
+    def test_upload_ova_file(self):
```
Comment (belwase) [inline tests/unit/vsphere/test_OVA.py:68]: what are you trying to test here ?  nothing is being test here

**Comment by belwase** [comment]
@samarthdd  build is also failing

**Comment by samarthdd** [comment]
@belwase Yes it was failing since there was issue with WEBhandler, Now it is fixed

**Comment by belwase** [comment]
Closing this since directory structure is changed, there is new issue for improving ova tests. https://github.com/k8-proxy/k8-vmware/issues/21

**Pull request closed without merging**